style.css

```diff
@@ -0,0 +1,581 @@
+}
+
+.intro h2 {
+  font-size: 28px;
+  margin-top: 15px;
+  font-weight: 800;
+}
+
+.intro h1 {
+  color: rgb(236 12 12 / 86%);
+  font-size: 32px;
+  line-height: 1.1;
+  margin: 0.5rem 0;
+  text-transform: uppercase;
+}
+
+.pros p {
+  width: 65%;
+}
+
+.intro p:last-child,
+.intro h2 {
+  color: #272a31;
+}
+
+.bg {
+  background-color: rgb(245 243 243 / 93%);
+  border: 2px solid white;
+  padding: 10px;
+  margin: 1rem 0;
+}
+
+.show {
+  right: 0;
+  display: block;
+}
+
+.how .fa-bars {
+  display: none;
+}
+
+.how .fa-xmark {
+  display: block;
+}
+
+#larger {
+  display: none;
+}
+
+.program {
+  padding: 2rem 1rem;
+  background-image: url(images/program-mobile.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.program h2 {
+  text-align: center;
+  text-transform: capitalize;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+}
+
+.pros {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 0;
+  padding: 5px;
+  background-color: rgba(255, 255, 255, 0.288);
+  transition: 0.3s linear;
+}
+
+.pros:hover {
+  box-shadow: 2px 0 10px 0 gray;
+}
+
+.pros .img {
+  display: flex;
+  align-items: center;
+  width: 30%;
+}
+
+.more button i {
+  color: #ec5242;
+}
+
+.pros .img i {
+  margin-right: 5px;
+  font-size: 25px;
+}
+
+.pros .img h4 {
+  text-transform: capitalize;
+  font-size: 16px;
+  color: #ec5242;
+  font-weight: 800;
+}
+
+.program .btn {
+  text-align: center;
+  margin: 10px 0;
+}
+
+.program button {
+  background-color: #ec5242;
+  padding: 1rem;
+  width: 75%;
+  border: none;
+  color: white;
+  font-size: 16px;
+  text-align: center;
+  font-family: lato, sans-serif;
+}
+
+
+
+
+/****************  featured speaker section  *****************************/
+
+.wrapper {
+  display: grid;
+  gap: 1rem;
+}
+
+.each {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  transition: 0.3s linear;
+}
+
+.each .img {
+  width: 32%;
+  position: relative;
+  padding: 5px;
+}
+
+.each .img img {
+  z-index: 99;
+  position: inherit;
+  height: 80%;
+  width: 100%;
+}
+
+.each .texts {
+  width: 63%;
+}
+
+.each .texts h3 {
+  text-transform: capitalize;
+}
+
+.each .texts h4 {
+  color: #ec5242;
+  text-transform: capitalize;
+  margin: 5px 0;
+}
+
+.each .texts p {
+  margin: 5px 0;
+  color: #272a31;
+}
+
+.backdrop {
+  position: absolute;
+  width: 100px;
+  height: 100px;
+  background-color: #ec5242;
+  top: -10px;
+  left: -5px;
+}
+
+.more button {
+  width: 100%;
+  background-color: transparent;
+  border: 1px solid rgb(78, 78, 78);
+  padding: 0.5rem;
+  font-size: 20px;
+  font-family: lato, sans-serif;
+  display: flex;
+  color: #272a31;
+  align-items: center;
+  justify-content: center;
+}
+
+.more span {
+  margin-right: 5px;
+  text-transform: uppercase;
+  font-size: 15px;
+}
+
+.sponsors {
+  display: none;
+  background-image: url(images/speaker.jpg);
+  background-size: cover;
+  color: white;
+}
```

Comment:
- [ ] Kindly use the same background from the template for the _**Sponsors Section**_. Using the same background would make your work identical to that used in the template. Please implement this for both desktop and mobile views on both the **_Home_** and _**About**_ Pages. 💻📱 

### Your design

![image](https://user-images.githubusercontent.com/77106962/210667732-333e5af1-656d-4b69-bb1d-0e2ef21f1090.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/210667913-1320ea52-5bc8-45f7-96de-be811261ffa9.png)